app/templates/service_update_audits.html

```diff
@@ -0,0 +1,154 @@
+
+<div class="page-container">
+    <div class="grid-row">
+        <div class="service-title">
+            {{ page_heading(today, "Activity for", "page-heading-smaller") }}
+        </div>
+    </div>
+    <div class="grid-row">
+        {% if form.errors|length >= 1 %}
+        <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+            <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+                There was a problem with your answer to the following questions
+            </h3>
+            {% for field_name, field_errors in form.errors|dictsort if field_errors %}
+            {% for error in field_errors %}
+            <a href="#example-textbox" class="validation-masthead-link">{{ form[field_name].label }}</a>
+            {% endfor %}
+            {% endfor %}
+        </div>
+        {% endif %}
+
+
+
+
+        <div class="grid-column third-column">
+        <form action="{{ url_for('.service_update_audits') }}" method="get" class="question">
+            <div class="govuk-option-select">
+                {{ form.audit_date.label(class="option-select-label") }}
+                {% if form.audit_date.errors %}
+                <p class="validation-message" id="error-email-address-textbox">
+                    {% for error in form.audit_date.errors %}{{ error }}{% endfor %}
+                </p>
+                {% endif %}
+                {{ form.audit_date(class="filter-field-text") }}
+            </div>
+
+            <div class="govuk-option-select">
+                <div class="container-head js-container-head">
+                    <div class="option-select-label">Show</div>
+                </div>
+                <div class="options-container">
+                    <div class="js-auto-height-inner">
+                        <label for="acknowledged-1">
+                            <input name="acknowledged" value="all" id="acknowledged-1" type="radio"
+                                   aria-controls=""
+                                   {% if acknowledged=="all" %} checked {% endif %}>
+                            All
+                        </label>
+                        <label for="acknowledged-2">
+                            <input name="acknowledged" value="true" id="acknowledged-2" type="radio"
+                                   aria-controls=""
+                                   {% if acknowledged=="true" %} checked {% endif %}>
+                            Acknowledged
+                        </label>
+                        <label for="acknowledged-3">
+                            <input name="acknowledged" value="false" id="acknowledged-3" type="radio"
+                                   aria-controls=""
+                                   {% if acknowledged=="false" %} checked {% endif %}>
+                            Not acknowledged
+                        </label>
+
+                    </div>
+                </div>
+            </div>
```

Comment:
I'm not sure if it's a big deal but I think the preferred way of using the frontend toolkit is via the provided templates now, so that markup changes can be rolled out more easily. cc/ @tombye @quis